app/src/main/java/org/sopt/alami/presentation/alarm/AlarmScreen.kt

```diff
@@ -0,0 +1,39 @@
+package org.sopt.alami.presentation.alarm
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.style.TextAlign
+import org.sopt.alami.core.designsystem.theme.AlarmiTheme
+
+@Composable
+fun AlarmRoute(
+    padding: PaddingValues
+) {
+    AlarmScreen(padding)
+}
+
+@Composable
+fun AlarmScreen(
+    padding: PaddingValues
+) {
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .background(AlarmiTheme.colors.black),
```

Comment:
```suggestion
        modifier = Modifier
            .fillMaxSize()
            .background(AlarmiTheme.colors.black)
            .padding(padding),
```


p1
미리 넣어놓으면 좋을 것 같아요~ 바텀바만큼 패딩을 사용해야하니ㅎㅎ